### ISSUE TITLE
Function find_partition() have no is_membership_fixed argument

### ISSUE THREAD

**Issue by CDPDisk** [vtraag/leidenalg#99]
I noticed that in your docs about [fixed nodes](https://leidenalg.readthedocs.io/en/latest/advanced.html#fixed-nodes) it says: 

> This can be done using the _is_membership_fixed_ argument of find_partition(), see 2 for some details.

And a sample program is provided. But after I viewed the code, I found that _find_partition()_ does not support the argument _is_membership_fixed_. If I were to use it, I'd have to give up _find_partition()_ and customize a new _find_partition()_ myself to use this argument. Although it is easy, I'd like to add the argument to find_partition() in the future version.

**Comment by vtraag** [comment]
Indeed, `find_partition` does not support this argument now right away. For more advanced uses, you should construct an `Optimiser` object and call `optimise_partition`, for which you can use the argument `is_membership_fixed`, e.g. for some `partition` and some list `fixed` indicating for each node whether its membership is fixed, you can call it like

```python
optimiser = la.Optimiser()
diff = optimiser.optimise_partition(partition, is_membership_fixed=fixed)
```

Your question is essentially whether `is_membership_fixed` can also be made available directly through `find_partition`?

**Comment by CDPDisk** [comment]
Yes, I hope `is_membership_fixed` can also be made available directly through find_partition. So fixing membership can be more easy.
